blocks/image-flush-bottom/image-flush-bottom.css

```
@@ -0,0 +1,34 @@
+div:first-child.image-flush-bottom-wrapper,
```

```suggestion
main .section.image-flush-bottom-container > .image-flush-bottom-wrapper
```

Let's keep this defined to the image flush bottom section/classes.